src/cohort.js

```diff
@@ -0,0 +1,96 @@
+class Cohort {
+  constructor() {
+    this.cohortList = []
+  }
+
+  createCohort(cohortName) {
+    const newCohort = { name: cohortName, students: [], capacity: 24 }
+    const findCohort = this.cohortList.find(
+      (cohort) => cohort.name === cohortName
+    )
+
+    if (findCohort) {
+      throw new Error('This Cohort already exists, please choose another name!')
+    }
+    if (!cohortName) {
+      throw new Error('Please give a name to your class!')
+    } else {
+      this.cohortList.push(newCohort)
+    }
```

Comment:
You don't need the else here, you can just push the cohort into the list and then return the list